### PR TITLE
doc/ceph-volume: refine encryption.rst

### PR DESCRIPTION
**Pull request by zdover23**
Improve the word choice and grammar of
doc/ceph-volume/lvm/encryption.rst.

Signed-off-by: Zac Dover <zac.dover@gmail.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/ceph-volume/lvm/encryption.rst [hunk 1]
```diff
@@ -4,42 +4,43 @@ Encryption
...
+LVM does, but it encrypts logical volumes in a way that  is consistent and
 robust.
 
 In this case, ``ceph-volume lvm`` follows these constraints:
 
-* Logical Volumes are encrypted, while their underlying PVs (physical volumes)
-  aren't
-* Non-LVM devices like partitions are also encrypted with the same OSD key
+* Logical Volumes (LVs) are encrypted, but their underlying physical volumes
+  (PVs) are not encrypted.
```
Comment (anthonyeleven) [inline doc/ceph-volume/lvm/encryption.rst:15]: I haven't worked a lot with `ceph-volume` but this is surprising and alarming.  Encryption-at-rest is the whole point of `--dmcrypt`.  I suspect though that the wording here is misleading, that what it means to say is that encryption is applied at the LVM level, and that the encrypted data is written to the underlying physical *devices*, vs using something like OPAL or SED.  I suggest checking with Neha or an OSD/RADOS authority for clarification here.

**Review by anthonyeleven — APPROVED** [review]
Non-blocking but strong suggestion

File: doc/ceph-volume/lvm/encryption.rst (hunk 1 above)
Comment (zdover23) [reply doc/ceph-volume/lvm/encryption.rst:15]: I think this is strong enough to be a blocker. @neha-ojha: is it true that the physical volumes that support logical volumes aren't encrypted when you use ``--dmcrypt``?

Comment (anthonyeleven) [reply doc/ceph-volume/lvm/encryption.rst:15]: Ack.  Really I don't know how they couldn't be, I strongly suspect this is misleading wording.

Comment (zdover23) [reply doc/ceph-volume/lvm/encryption.rst:15]: I've spoken to Josh about it, and we're going to cut this out entirely. It's just confusing.

Comment (anthonyeleven) [reply doc/ceph-volume/lvm/encryption.rst:15]: Agreed.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/49792 - Quincy backport
https://github.com/ceph/ceph/pull/49793 - Pacific backport

**Comment by zdover23** [comment]
jenkins render docs

**Comment by zdover23** [comment]
jenkins retest this please

**Pull request merged**